config/middleware/receive-message/map-request-params.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const configVars = {};
+
+configVars.client = 'gambit-conversations';
+configVars.containerProperty = 'body';
+configVars.emojiStripParam = 'text';
+configVars.lowercaseParam = 'text';
+configVars.paramsMap = {
+  keyword: 'keyword',
+  broadcastId: 'broadcast_id',
+  text: 'incoming_message',
+  mediaUrl: 'incoming_image_url',
```

Comment:
This was the culprit: https://github.com/DoSomething/gambit/blob/4.6.0/lib/middleware/map-request-params.js#L78 -- we don't need to call `parseCampaignId` from the `receive-message` route.

https://github.com/DoSomething/gambit/pull/945